### ISSUE TITLE
Bot transferred all pokemon

### ISSUE THREAD

**Issue by Supergod99** [NECROBOTIO/NecroBot#2352]
Every Pokemon that was below "KeepMinCp: 2050" was deleted, including pokemon on the not to transfer list, and pokemon that were my best of that pokemon but below the 2050 CP.

**Comment by deathart** [comment]
Change
"TransferWeakPokemon": false,

**Comment by Supergod99** [comment]
Just tried that, and it just deleted my 2105 Machamp, which is over the limit, and was my best machamp I had....

**Comment by actisys** [comment]
"PrioritizeIvOverCp": true,  <-- set to false, i think your setting only keep better IV rather than CP.

**Comment by mennas** [comment]
not just that, even from my filter transfer list is transfered WTF???!!!!
im not gonna use this bot till this fix

**Comment by mennas** [comment]
i set on filter list to keep 3 poke
    "Vaporeon": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3,
      "Moves": [],
      "KeepMinOperator": "or"

but it transfer 1 with highest CP around 80's IV WTFFF

**Comment by Skifterok** [comment]
Same thing here. Was shocked out when all my -1000 pokemons dissappeared(((

**Comment by mennas** [comment]
lucky i was watching the bot, so i could stop it in the middle
otherwise it will ruin my pokemon

**Comment by mennas** [comment]
i also set this

    "Machamp": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2

keep 2, but it transfer 2 of my machamp
just f**k

**Comment by Skifterok** [comment]
btw, all happened after changing keep min duplicate pok

**Comment by mennas** [comment]
listen, i set min dup to 1 as global, and add special rule at filter transfer. just before, i had 2 machamp with 2K cp and not so bad IV before, i run the bot, watch it a while, and it destroy days of hunt, quickly i stop it, and i check on the game by phone, f**k all ruined

**Comment by mennas** [comment]
the bot broke all this fukin  rule : 

  "PokemonsTransferFilter": {
    "Pidgeot": {
      "KeepMinCp": 2000,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Alakazam": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 98.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Starmie": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3
    },
    "Aerodactyl": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Gyarados": {
      "KeepMinCp": 2600,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3
    },
    "Slowbro": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Rapidash": {
      "KeepMinCp": 2400,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1
    },
    "Wigglytuff": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Clefable": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1
    },
    "Venusaur": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Charizard": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Blastoise": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Golduck": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Arcanine": {
      "KeepMinCp": 2600,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Magikarp": {
      "KeepMinCp": 205,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Machamp": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Farfetchd": {
      "KeepMinCp": 2000,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1
    },
    "Muk": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1
    },
    "Exeggutor": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3
    },
    "Kangaskhan": {
      "KeepMinCp": 2000,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1
    },
    "MrMime": {
      "KeepMinCp": 2000,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1
    },
    "Lapras": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Ditto": {
      "KeepMinCp": 1800,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1
    },
    "Vaporeon": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3
    },
    "Jolteon": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3
    },
    "Flareon": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3
    },
    "Eevee": {
      "KeepMinCp": 1250,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Porygon": {
      "KeepMinCp": 1850,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Snorlax": {
      "KeepMinCp": 2750,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 4
    },
    "Articuno": {
      "KeepMinCp": 1000,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Zapdos": {
      "KeepMinCp": 1000,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Moltres": {
      "KeepMinCp": 1000,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Dragonite": {
      "KeepMinCp": 2900,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 5
    },
    "Dratini": {
      "KeepMinCp": 1000,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Mew": {
      "KeepMinCp": 1000,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 2
    },
    "Mewtwo": {
      "KeepMinCp": 100,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 2
    }
  },

**Comment by darkzurvan** [comment]
I lost about half my pokemon this morning before I realized what was happening. Changing the "TransferWeakPokemon" to false worked for me, I'm not sure why it didn't for you. I'm sorry. Risks we take I guess. :(

**Comment by Skifterok** [comment]
transfer weak flood my bag with very low cp poks

**Comment by mennas** [comment]
they should give a proper guide on setting before release it, more detail, more feature, more complex more fucked up. people hope for it to be better, maybe it better but without a proper guide, its just like giving 5 years old kid a gun

**Comment by darkzurvan** [comment]
@Skifterok Yeah there's another issue where it won't transfer duplicates, I'm grappling with that right now and I can't figure it out. I think it's a hardcoded issue that we just need to wait on for a new release.

@mennas I can't judge the devs and their methods because I know nothing about how their system works, but the little un-numbered updates are a little irritating, adding new parameters and stuff without warning or updating the settings guide until much later. If I were a donator I'd probably be a lot angrier but I'm just a leech with no money.

I think once they make a stable release I'm going to change the "AutoUpdate" parameter to false for a while, so it won't happen again.

**Comment by NurAdam** [comment]
@mennas  ur program seem to bit old. Latest release have improved argument command. I updated the bot just now. Will give feed back after merging my config. here some change i seen. 

"Vaporeon": {
      "KeepMinCp": 1500,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Jolteon": {
      "KeepMinCp": 1500,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Flareon": {
      "KeepMinCp": 1500,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or"
    },

**Comment by mennas** [comment]
wrong paste, same thing



  "PokemonsToIgnore": [],
  "PokemonsTransferFilter": {
    "Pidgeot": {
      "KeepMinCp": 2000,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Alakazam": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 98.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Starmie": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Aerodactyl": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Gyarados": {
      "KeepMinCp": 2600,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Slowbro": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Rapidash": {
      "KeepMinCp": 2400,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Wigglytuff": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Clefable": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Venusaur": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Charizard": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Blastoise": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Golduck": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Arcanine": {
      "KeepMinCp": 2600,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Magikarp": {
      "KeepMinCp": 205,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Machamp": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Farfetchd": {
      "KeepMinCp": 2000,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Muk": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Exeggutor": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Kangaskhan": {
      "KeepMinCp": 2000,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "MrMime": {
      "KeepMinCp": 2000,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Lapras": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Ditto": {
      "KeepMinCp": 1800,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Vaporeon": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Jolteon": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Flareon": {
      "KeepMinCp": 2500,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 3,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Eevee": {
      "KeepMinCp": 1250,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Porygon": {
      "KeepMinCp": 1850,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Snorlax": {
      "KeepMinCp": 2750,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 4,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Articuno": {
      "KeepMinCp": 1000,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Zapdos": {
      "KeepMinCp": 1000,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Moltres": {
      "KeepMinCp": 1000,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Dragonite": {
      "KeepMinCp": 2900,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 5,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Dratini": {
      "KeepMinCp": 1000,
      "KeepMinIvPercentage": 100.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Mew": {
      "KeepMinCp": 1000,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    },
    "Mewtwo": {
      "KeepMinCp": 100,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 2,
      "Moves": [],
      "KeepMinOperator": "or"
    }

**Comment by NurAdam** [comment]
i see. geez now im in doubt to run the bot. xd

**Comment by IamAii** [comment]
I wanted to keep my Eevee's and It still transfer's them. On a side note I have not lost my good higher level CP's (yet)...

I added Eevee to the "PokemonsNotToTransfer" list, and the 
},
    "Eevee": {
      "KeepMinCp": 450,
      "KeepMinIvPercentage": 25.0,
      "KeepMinDuplicatePokemon": 10,
      "Moves": [],
      "KeepMinOperator": "or"
    },

And it still Transfer's everything over those settings.  I have not had this program long, so I am still getting use to it, but damn It shouldnt transfer.

**Comment by mennas** [comment]
as deathart said, turn transferweakpokemon to false, to disable the shit

**Comment by mennas** [comment]
i have 2500 CP with IV round 80's of IV, and its gone
Vaporeon,34,2500,84.70%,WaterGunFast,AquaTail

**Comment by NurAdam** [comment]
i think the ["KeepMinOperator": _"or"_ ]
seem odd. why the [OR] need to be inside quotation mark.  all the false/true doesn't need to be inside "" (quotation mark).

**Comment by Supergod99** [comment]
"TranslationLanguageCode": "en",
  "AutoUpdate": true,
  "TransferConfigAndAuthOnUpdate": true,
  "StartupWelcomeDelay": false,
  "AmountOfPokemonToDisplayOnStart": 10,
  "ShowPokeballCountsBeforeRecycle": true,
  "AutomaticallyLevelUpPokemon": false,
  "AmountOfTimesToUpgradeLoop": 5,
  "GetMinStarDustForLevelUp": 5000,
  "LevelUpByCPorIv": "iv",
  "UpgradePokemonCpMinimum": 1000.0,
  "UpgradePokemonIvMinimum": 95.0,
  "UpgradePokemonMinimumStatsOperator": "and",
  "DisableHumanWalking": false,
  "DefaultAltitude": 10.0,
  "DefaultLatitude": 43.037101,
  "DefaultLongitude": -78.696367,
  "WalkingSpeedInKilometerPerHour": 120.0,
  "MaxSpawnLocationOffset": 10,
  "DelayBetweenPlayerActions": 10,
  "DelayBetweenPokemonCatch": 10,
  "DumpPokemonStats": false,
  "EvolveAboveIvValue": 95.0,
  "EvolveAllPokemonAboveIv": false,
  "EvolveAllPokemonWithEnoughCandy": false,
  "EvolveKeptPokemonsAtStorageUsagePercentage": 90.0,
  "KeepPokemonsThatCanEvolve": false,
  "KeepMinCp": 2050,
  "KeepMinIvPercentage": 90.0,
  "KeepMinOperator": "or",
  "PrioritizeIvOverCp": false,
  "KeepMinDuplicatePokemon": 1,
  "UseGpxPathing": false,
  "GpxFile": "GPXPath.GPX",
  "VerboseRecycling": true,
  "RecycleInventoryAtUsagePercentage": 90.0,
  "UseEggIncubators": true,
  "UseLuckyEggConstantly": false,
  "UseLuckyEggsMinPokemonAmount": 30,
  "UseLuckyEggsWhileEvolving": false,
  "UseIncenseConstantly": false,
  "UseBerriesMinCp": 1000,
  "UseBerriesMinIv": 90.0,
  "UseBerriesBelowCatchProbability": 0.2,
  "UseBerriesOperator": "and",
  "UseSnipeOnlineLocationServer": true,
  "UseSnipeLocationServer": false,
  "SnipeLocationServer": "localhost",
  "SnipeLocationServerPort": 16969,
  "MinPokeballsToSnipe": 20,
  "MinPokeballsWhileSnipe": 0,
  "MinDelayBetweenSnipes": 60000,
  "SnipingScanOffset": 0.003,
  "SnipeAtPokestops": false,
  "SnipeIgnoreUnknownIv": false,
  "UseTransferIvForSnipe": false,
  "RenamePokemon": false,
  "RenameOnlyAboveIv": true,
  "RenameTemplate": "{1}_{0}",
  "MaxPokeballsPerPokemon": 6,
  "MaxTravelDistanceInMeters": 1000,
  "TotalAmountOfPokeballsToKeep": 120,
  "TotalAmountOfPotionsToKeep": 80,
  "TotalAmountOfRevivesToKeep": 60,
  "TotalAmountOfBerriesToKeep": 50,
  "UseGreatBallAboveCp": 1000,
  "UseUltraBallAboveCp": 1250,
  "UseMasterBallAboveCp": 1500,
  "UseGreatBallAboveIv": 85,
  "UseUltraBallAboveIv": 95,
  "UseGreatBallBelowCatchProbability": 0.2,
  "UseUltraBallBelowCatchProbability": 0.1,
  "UseMasterBallBelowCatchProbability": 0.05,
  "TransferWeakPokemon": false,
  "TransferDuplicatePokemon": true,
  "TransferDuplicatePokemonOnCapture": true,
  "FavoriteMinIvPercentage": 95.0,
  "AutoFavoritePokemon": false,
  "UsePokemonToNotCatchFilter": false,
  "UsePokemonSniperFilterOnly": false,
  "WebSocketPort": 14251,

Just transferred my best dratini's and dragonair's, even after I put them on the PokemonNotToTransfer List. Can someone please tell me what I did wrong here?

**Comment by mennas** [comment]
make your best as favorite, if it still transfered ask the dev

**Comment by darkzurvan** [comment]
I've seen some evidence in other posts that the KeepMin parameters MIGHT have too much authority and override the nottotransfer list. So:

"KeepMinCp": 2050,
"KeepMinIvPercentage": 90.0,

 MIGHT have been set too high. There might have been a problem with your list entries too, not under normal circumstances but just under this current build.

**Comment by Supergod99** [comment]
"KeepMinCp": 1250,

No luck with this setting either, transferred a 1500 slowbro I had.

**Comment by Supergod99** [comment]
Im not using this bot im afraid it'll transfer all of my pokemon, i will just wait for a fixed build.

**Comment by mennas** [comment]
1250 will make your inventory full

**Comment by Triky313** [comment]
Yes, my inventory is also full. No Chance to transfer anyone with the Bot.

**Comment by Georgexp** [comment]
[00:57:38] (TRANSFERED) NidoranF        - CP: 448  IV: 21.81%   [Best CP: 561  IV: 56.44%] (Candi
[00:57:45] (TRANSFERED) NidoranF        - CP: 561  IV: 56.44%   [Best CP: 33  IV: 45.18%] (Candie
[00:57:45] (TRANSFERED) Spearow - CP: 62  IV: 51.38%   [Best CP: 420  IV: 60.13%] (Candies: 261)
[00:57:46] (TRANSFERED) NidoranF        - CP: 10  IV: 38.99%   [Best CP: 33  IV: 45.18%] (Candies
[00:57:46] (TRANSFERED) Eevee   - CP: 513  IV: 66.16%   [Best CP: 563  IV: 83.13%] (Candies: 579)
[00:57:48] (TRANSFERED) NidoranF        - CP: 33  IV: 45.18%   [Best CP: 33  IV: 45.18%] (Candies
[00:57:48] (TRANSFERED) Krabby  - CP: 78  IV: 79.37%   [Best CP: 400  IV: 92.91%] (Candies: 222)
[00:57:49] (TRANSFERED) Spearow - CP: 420  IV: 60.13%   [Best CP: 420  IV: 60.13%] (Candies: 261)

**Comment by Caliboss** [comment]
i lost a dragonite and charizard both around 2000 cp :( along with about half of my pokemon .. i liked version 4 lol.. im staying on version 5 till some of this transfer stuff is fixed.. only testing 1 account right now on version 6.3 .. hopefully it goes well with no errors thru the night.

mine yesterday ran for 14 hours straight with 0 errors. this afternoon ive had a error every 2 hours.. im trying to fix it my self but no luck so far. still doing testing

**Comment by darkzurvan** [comment]
@Caliboss I just tried 0.6.3 and I must say you're right, it's working pretty well. Don't know if it's evolving but it looks like it's transferring the right ones. I guess my biggest problem was trusting auto-update to get me out of the unstable release.
